ui/pages/confirm-transaction-base/confirm-transaction-base.container.js

```diff
@@ -63,8 +71,10 @@ import {
 import { getGasLoadingAnimationIsShowing } from '../../ducks/app/app';
 import { isLegacyTransaction } from '../../helpers/utils/transactions.util';
 import { CUSTOM_GAS_ESTIMATE } from '../../../shared/constants/gas';
+
 ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
 import { getAccountType } from '../../selectors/selectors';
```

Comment:
I'm not sure exactly what principles Brad has been encouraging in reviews so far, but I would say we don't need to add code fences unless necessary. I think these imports, and their use below, can safely be added to this file without fences.

In fact, I think we could remove all fences from this file. Perhaps a selector, or multiple selectors, could be created that contain the mmi specific logic. And they could be named to make it clear they are mmi specific. Then the appropriate properties can be returned from the `map*ToProps` functions, and the component can decide which properties to use.

In general, I think the fewer code fences the better, and this strikes me as the sort of file where we can safely avoid any code fences.

Reply:
Hey, @danjm, thank you for the provided feedback.

About code fencing, at least, I didn't know that. Please let me know if I'm wrong @zone-live, but we have added as many code fences as necessary.

I will remove those fences from this file, and I'll have it in mind for future PRs but keep in mind that all the PRs we have merged "may" not be done correctly.

Reply:
I updated the code. Please, let me know if the current approach is the best.